### PR TITLE
provide own buffer for images

### PR DESCRIPTION
**Pull request by oheil**

### CODE REVIEW

**Comment by cormullion** [comment]
Thanks!

File: docs/src/howto/livegraphics.md [hunk 1]
```diff
@@ -95,6 +95,115 @@ end
 
 clock(ColorSchemes.klimt)
 ```
+Another example of using Luxor with MiniFB as the display window, but without the `@play` macro.
+This allows for interactive graphic commands on the REPL instantly displayed in the window:
+```
+using MiniFB, Luxor, Colors
+
+function windowUpdateTask(window,buffer,showFPS=false)
```
[the remark this answers is not in the document]
Comment (oheil) [reply docs/src/howto/livegraphics.md:103]: If not ok, no problem, will change. Just give me a note...

[the remark this answers is not in the document]
Comment (cormullion) [reply docs/src/howto/livegraphics.md:103]: That's true, camelCase isn't used much in this repo. I'll change it at some point, it's a docs example, fortunately... :)

[the remark this answers is not in the document]
Comment (oheil) [reply docs/src/howto/livegraphics.md:103]: No problem, I am already preparing the change. small PR will follow

[the remark this answers is not in the document]
Comment (hyrodium) [reply docs/src/howto/livegraphics.md:103]: Thanks for the fix!

**Pull request merged**